Temaors/unit tests

temaors opened the pull request.

nikita-ermolaev-coral committed:
added separate interface for UserTask model

nikita-ermolaev-coral committed:
added separate interface for UserTask model

nikita-ermolaev-coral committed:
added UserTaskFixture class

nikita-ermolaev-coral committed:
added UserTaskFixture class

nikita-ermolaev-coral committed:
remove async in IUserTaskRepository

nikita-ermolaev-coral committed:
remove async in IUserTaskRepository

nikita-ermolaev-coral committed:
added tests for UserTaskRepository

nikita-ermolaev-coral committed:
added tests for UserTaskRepository

nikita-ermolaev-coral committed:
added tests for TaskTrackRepository

nikita-ermolaev-coral committed:
added tests for TasStateRepository without implementation

nikita-ermolaev-coral committed:
added implementation for TaskStateRepository tests

nikita-ermolaev-coral committed:
merged temaors's repositories with current branch

nikita-ermolaev-coral committed:
added tests for UserTaskRepository

nikita-ermolaev-coral committed:
added AbstractRepositoryFixture abstract class for fixtures

nikita-ermolaev-coral committed:
added tests for UserProfileRepository

temaors committed:
added unit AbstractRepositoryFixture.cs,
added unit tests for TaskRepository

temaors committed:
added unit AbstractRepositoryFixture.cs,
added unit tests for TaskRepository

temaors committed:
fixed tests

temaors committed:
fixed tests

nikita-ermolaev-coral committed:
removed tabulation

temaors committed:
synchronized with AbstractRepositoryFixture

nikita-ermolaev-coral reviewed (approved):
good job

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/TaskStateRepositoryTests.cs:
```diff
@@ -0,0 +1,147 @@
+        }
+
+        [Fact]
+
```
Remove the space

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs:
```diff
@@ -0,0 +1,157 @@
+        }
+
+        [Fact]
+
```
remove space

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs:
```diff
@@ -0,0 +1,157 @@
+        }
+
+        [Fact]
+
```
remove space

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/TaskTrackRepositoryTest.cs:
```diff
@@ -0,0 +1,157 @@
+            // Act & Assert
+            await Assert.ThrowsAsync<ArgumentException>(() => _fixture.Repository.Delete(id));
+        }
+        public void Dispose()
```
add space before the method

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs:
```diff
@@ -0,0 +1,207 @@
+        }
+
+        [Fact]
+
```
remove the space

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs:
```diff
@@ -0,0 +1,207 @@
+        }
+
+        [Fact]
+
```
remove the space

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/UserProfileRepositoryTests.cs:
```diff
@@ -0,0 +1,207 @@
+            // Act & Assert
+            await Assert.ThrowsAsync<ArgumentException>(() => _fixture.Repository.Delete(id));
+        }
+        public void Dispose()
```
add space before the method

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs:
```diff
@@ -0,0 +1,160 @@
+        }
+
+        [Fact]
+
```
remopve the space

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs:
```diff
@@ -0,0 +1,160 @@
+        }
+
+        [Fact]
+
```
remove the space

dbogdan2000 commented on DIMS-Core/DIMS-Core.Tests/Repositories/UserTaskRepositoryTests.cs:
```diff
@@ -0,0 +1,160 @@
+            // Act & Assert
+            await Assert.ThrowsAsync<ArgumentException>(() => _fixture.Repository.Delete(id));
+        }
+        public void Dispose()
```
add space before the method

temaors committed:
fixed spaces int test's files
changed sync operation to async in DirectionService.cs and UserProfileService.cs

temaors committed:
Merge branch 'main' into temaors/unit-tests
